feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/ExecutionTracker.kt

```diff
@@ -38,15 +39,15 @@ internal class ExecutionTracker @Inject constructor(
             if (currentLifecycleOwnerId == null || enoughTimePassedSinceLastLock()) {
                 currentLifecycleOwnerId = ownerId
                 timestamp = timeHelper.now()
-                Simber.i("Lifecycle owner [$ownerId] is set as main executor")
+                Simber.tag(ORCHESTRATION).i("Lifecycle owner [$ownerId] is set as main executor")
             } else {
-                Simber.i("Main executor already set, ignoring Lifecycle owner [$ownerId]")
+                Simber.tag(ORCHESTRATION).i("Main executor already set, ignoring Lifecycle owner [$ownerId]")
             }
         } else if (event == Lifecycle.Event.ON_DESTROY) {
             if (currentLifecycleOwnerId == ownerId) {
                 currentLifecycleOwnerId = null
                 timestamp = Timestamp(0)
-                Simber.i("Lifecycle owner [$ownerId] removed from main executor")
+                Simber.tag(ORCHESTRATION).i("Lifecycle owner [$ownerId] removed from main executor")
```

Comment:
Are these actually useful for anything?

Reply:
It was useful when we were dealing with the double session bug. Since it seems to be fixed, we could remove this.

Reply:
Ah, OK. If they were used in the past then let's keep them.